src/api/operator/operatorApi.ts

```diff
@@ -265,17 +147,12 @@ export async function getTxOrders(params: GetTxOrdersParams): Promise<RawOrder[]
  * Both filters cannot be used at the same time
  */
 export async function getTrades(params: GetTradesParams): Promise<RawTrade[]> {
-  const { networkId, owner = '', orderId = '' } = params
-
-  if (owner && orderId) {
-    throw new Error('Cannot use both `owner` and `orderId` filters at the same time')
-  }
-
-  console.log(`[getTrades] Fetching trades on network ${networkId} with filters: owner=${owner} orderId=${orderId}`)
+  const { networkId, owner = '' } = params
+  const cowInstance = COW_SDK[networkId]
 
-  const queryString = `/trades/` + buildSearchString({ owner, orderUid: orderId })
```

Comment:
we haven't included the `orderId` in the `getTrades` method and we should. Is the Explorer making use of the `orderId` somewhere?